packages/formatter-html/src/_locales/en/messages.json

```diff
@@ -31,6 +31,10 @@
         "description": "Text for the expand all button",
         "message": "expand all"
     },
+    "furtherReading": {
+        "description": "Header for further reading in hints with external documentation",
+        "message": "Further Reading"
```

Comment:
I don't think we have to solve this now, but we may want to move this string to `@hint/utils-i18n` so we can share it between formatters and with the browser extension (and avoid localizing the string multiple times).

Reply:
Do we support that?

Reply:
We're not explicitly exporting a helper to get it at the moment, but `import getMessage as getSharedMessage from '@hint/utils-i18n/dist/src/i18n.import';` then using `getSharedMessage("furtherReading")` should work. 

This would be better still if we just export a `getSharedMessage` helper from `@hint/utils-i18n`. Currently we only share category names so we exported a much more scoped `getCategoryName` helper.